opal/mca/common/ucx/common_ucx_wpool_int.h

```diff
@@ -5,7 +5,7 @@
 #include "common_ucx.h"
 #include "common_ucx_wpool.h"
 
-static int _tlocal_ctx_connect(_ctx_record_t *ctx_rec, int target);
+static int _tlocal_ctx_connect(opal_common_ucx_wpmem_t *mem, _ctx_record_t *ctx_rec, int target);
```

Comment:
Is this change in the signature still needed? The call at https://github.com/open-mpi/ompi/pull/10709/files#diff-4277b36854f67c21224bff1f16f6aa090af4e3905c9e00ce96dc735f05c90200R736 doesn't pass the extra `opal_common_ucx_wpmem_t*` pointer...